cmd/stop.go

```diff
@@ -32,14 +32,18 @@ var stopCmd = &cobra.Command{
 		Info.log("Stopping development environment")
 		projectDir := getProjectDir()
 		imageName := fmt.Sprintf("%s-dev", filepath.Base(projectDir))
-
-		dockerStop(imageName)
+		if containerName == "" {
+			dockerStop(imageName)
```

Comment:
containerName will always be set, you can remove this if check and the imageName and always use containerName